src/pages/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -0,0 +1,83 @@
+import { ArrowBackIcon, ChevronRightIcon, CopyIcon } from '@chakra-ui/icons';
+import {
+  Box,
+  Input,
+  InputGroup,
+  InputRightElement,
+  Text,
+  Tooltip,
+  useBreakpointValue,
+  useToast,
+} from '@chakra-ui/react';
+import { useRouter } from 'next/router';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface DashboardHeaderProps {
+  pageTitle: string;
+}
+
+const DashboardHeader: React.FC<DashboardHeaderProps> = function Dashboard({ pageTitle }) {
+  const router = useRouter();
+  const { t } = useTranslation('dashboard');
+  const [apiKey, setApiKey] = useState('');
+  const toast = useToast();
+  const ml = useBreakpointValue({ base: 10, lg: 270 });
+  const pageHeaderDisplay = useBreakpointValue({ base: 'block', lg: 'flex' });
+
+  const handleGoBack = () => {
+    router.back();
+  };
+
+  const handleCopyApiKey = () => {
+    setApiKey('example-api-key');
+    navigator.clipboard.writeText(apiKey);
+    toast({
+      title: t('API key copied'),
+      description: t('Your API key has been copied to your clipboard.'),
+      status: 'success',
+      duration: 5000,
+      isClosable: true,
+    });
+  };
+
+  return (
+    <>
+      <Box ml={ml} pt={5} display="flex">
```

Comment:
it's not clear why you need such drastically different `ml` values. If you really need to have different values, you can do the following
```suggestion
      <Box ml={{ base: 10, lg: 270 }} pt={5} display="flex">
```

but i have a feeling you don't need to do this and there's an alternative way of achieving the same margin effect while not needing two very different values